### ISSUE TITLE
HOWTO -> work with r issue

### ISSUE THREAD

**Issue by hgputnam** [UCSF-CBI/c4#57]
The article gives an example of library nlme. The example says we can find this in the /c4/software stack and we can for v4.03 which the example uses. It is not, however, available for 4.1.1:

```
$ ls -l /software/c4/cbi/software/R-4.1.1-gcc8/lib64/R/library/
total 0
dr-xr-xr-x  7 henrik cbc 147 Aug 16 07:00 base
dr-xr-xr-x  6 henrik cbc 132 Aug 16 07:00 compiler
dr-xr-xr-x  6 henrik cbc 135 Aug 16 07:00 datasets
dr-xr-xr-x  8 henrik cbc 164 Aug 16 07:00 graphics
dr-xr-xr-x 11 henrik cbc 209 Aug 16 07:00 grDevices
dr-xr-xr-x  8 henrik cbc 163 Aug 16 07:00 grid
dr-xr-xr-x  7 henrik cbc 148 Aug 16 07:00 methods
dr-xr-xr-x  8 henrik cbc 163 Aug 16 07:00 parallel
dr-xr-xr-x  7 henrik cbc 148 Aug 16 07:00 splines
dr-xr-xr-x  9 henrik cbc 230 Aug 16 07:00 stats
dr-xr-xr-x  6 henrik cbc 132 Aug 16 07:00 stats4
dr-xr-xr-x  9 henrik cbc 180 Aug 16 07:00 tcltk
dr-xr-xr-x  7 henrik cbc 148 Aug 16 07:00 tools
dr-xr-xr-x 21 henrik cbc 316 Aug 16 07:00 translations
dr-xr-xr-x 10 henrik cbc 218 Aug 16 07:00 utils
```

Just a bit confusing for. the uninitiated. Can we pick a different library example that will exist for all R versions?

**Comment by HenrikBengtsson** [comment]
I've updated https://www.c4.ucsf.edu/howto/r.html to use **datasets** instead.  The reason why **nlme** is no longer installed by default in R (>= 4.1.0), is that I decided to _not_ install the 10-20, so called, "recommended" packages globally.  Instead, each user will have to install those themself, which will often happen automatically in the background when other packages are installed.  The reason for this change is to avoid confusing, misleading warnings on not being able to update those.

FWIW, there's a lot of workarounds on that R help page that are no longer needed for R (>= 4.1.0).  This is because I know set R to use a more modern gcc compiler when the `r` module is loaded.
